cli/cli.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Novo Nordisk Foundation Center for Biosustainability, DTU.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__all__ = ['main']
+
+import warnings
+warnings.simplefilter("ignore", UserWarning)
+
+import multiprocessing
+import pickle
+import sys
+
+import click
+import pandas
+
+pandas.options.display.width = None
+pandas.options.display.max_rows = 30
+
+import logging
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel('WARNING')
+
+cameo_logger = logging.getLogger('cameo')
+cameo_logger.setLevel('WARNING')
+
+inspyred_logger = logging.getLogger('inspyred.ec')
+inspyred_logger.setLevel('WARNING')
+
+OUTPUT_WRITER = {
+    "xlsx": lambda df, path: df.to_excel(path),
+    "csv": lambda df, path: df.to_csv(path, sep=","),
+    "tsv": lambda df, path: df.to_csv(path, sep="\t"),
+}
+VALID_OUTPUT_FORMATS = list(OUTPUT_WRITER.keys())
+VALID_OUTPUT_FORMATS.append('pickle')
+
+
+def print_version(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    from cameo import __version__
+    click.echo(__version__)
+    ctx.exit()
+
+
+@click.group()
+@click.option('--version', is_eager=True, callback=print_version, default=False, is_flag=True,
+              help='Display version information.')
+def main(version):
+    """cameo command line interface"""
+    pass
+
+
+@main.command()
+@click.option('-o', '--output', default='-', multiple=True, type=click.Path(),
+              help='Output filename. Multiple output files can be provided (pair with respective format options).')
+@click.option('-f', '--format', multiple=True, default=['csv'],
+              type=click.Choice(VALID_OUTPUT_FORMATS), help='Output file format (default csv).')
+@click.option('-h', '--host', default=['ecoli'], multiple=True,
```

Comment:
Default in the documentation doesn't match the default value. Can I pass multiple --host argument? I cannot figure it out from click documentation.